web/src/app/shared/components/tree/tree.component.ts

```diff
@@ -92,19 +87,25 @@ export class TaskanaTreeComponent implements OnInit, AfterViewChecked, OnDestroy
       this.action = action;
     });
 
-    this.selectedClassificationId$.pipe(takeUntil(this.destroy$)).subscribe(selectedClassificationId => {
-      this.selectNodeId = typeof selectedClassificationId !== 'undefined' ? selectedClassificationId : undefined;
-      if (typeof this.tree.treeModel.getActiveNode() !== 'undefined') {
-        if (this.tree.treeModel.getActiveNode().data.classificationId !== this.selectNodeId) {
-          this.selectNode(this.selectNodeId);
+    const classificationCopy$: Observable<TreeNodeModel[]> = this.classifications$.pipe(
+      filter(classifications => typeof (classifications) !== 'undefined'),
+      tap(classifications => {
+        this.classifications = classifications.map(this.classificationsDeepCopy.bind(this));
+      })
+    );
+
+    combineLatest(this.selectedClassificationId$, classificationCopy$).pipe(takeUntil(this.destroy$))
```

Comment:
why are you not using classificationCopy$?

Reply:
I am using classificationCopy$, see the code snippet you marked

Reply:
I only see the selectedClassificationId in the subscribe statement. Or am I missing something?

Reply:
Ah I understand. The classifications are set with "tap" in the classificationCopy$ (line 93), that is why I don't need the classifications here...